src/hotspot/os_cpu/linux_ppc/os_linux_ppc.hpp

```diff
@@ -32,4 +32,10 @@
   // Note: Currently only used in 64 bit Windows implementations
   static bool register_code_area(char *low, char *high) { return true; }
 
+#ifndef VM_LITTLE_ENDIAN
```

Comment:
I'd prefer using #ifndef ABI_ELFv2 because it's directly related to the ABI, not to the endianess. It is used to control function descriptor related code usage in elfFile.cpp. It'd be great if you could use HAVE_FUNCTION_DESCRIPTORS there, too.

Reply:
Makes sense. I changed it to use ABI_ELFv2.